app/API/utilities/validator.py

```diff
@@ -1,64 +1,129 @@
+import re
+from datetime import datetime
+
 def validate_meetups(args):
     """validate meetup details"""
     try:
-        if args["location"] == '' or args["topic"] == '':
-            return {'error': 'Fields cannot be left empty'}, 401
-        elif( args["location"]. isdigit()) or ( args["topic"]. isdigit()) :
-            return {"error":"The fields should be described in words"},401
-        else:
-            return "valid"
+        if any([(arg.strip() == '') for arg in args]):
+            return{
+                "status": 401,
+                "error": "Fields cannot be left empty"
+                }, 401
+        elif( args["location"].isdigit()) or ( args["topic"].isdigit()) :
+            return{
+                "status": 401,
+                "error": "The fields should be described in words"
+                }, 401
+        dates = args['happeningOn']
+        try:
+            happenning = datetime.strptime(dates, "%d/%m/%Y")
+        except ValueError as exception:
+            return {}
+        days = (happenning - datetime.utcnow()).days
+        if days < 0:
+            return{
+                "status": 401,
+                "error": "Your happening date must be ahead of date of meetup posting"
```

Comment:
```suggestion
                "error": "A meetup cannot be scheduled in the past."
```

Consider changing this error message to a more understandable version as suggested